lib/experimental/Widgets/Content/TwoColumnsList/index.tsx

```diff
@@ -1,28 +1,34 @@
-import { forwardRef } from "react"
+import { forwardRef, ReactNode } from "react"
 
 interface TwoColumnsItemType {
   title: string
-  info: string
+  info: string | ReactNode
 }
 
 interface TwoColumnsListType {
+  title?: string
   list: TwoColumnsItemType[]
 }
 
 const Item = ({ title, info }: TwoColumnsItemType) => (
-  <>
-    <div className="line-clamp-2 text-f1-foreground-secondary">{title}</div>
-    <div className="font-medium">{info}</div>
-  </>
+  <div className="flex items-center justify-between">
+    <div className="flex text-f1-foreground-secondary">{title}</div>
```

Comment:
Consider switching to a paragraph to stay semantic, or maybe it should even be an item list.

I also dropped the `flex` since there are no block children inside
```suggestion
    <p className="text-f1-foreground-secondary">{title}</p>
```